### ISSUE TITLE
Decoder is failing to decode &rsquo;

### ISSUE THREAD

**Issue by spacecowboy** [MohamedRejeb/Ksoup#25]
Example test

```
class KsoupTest {
    @Test
    fun rsquo() {
        val text = "laptop, it&rsquo;s probably"
        val expected = "laptop, it’s probably"

        assertEquals(expected, KsoupEntities.decodeHtml(text))
    }
}
```

Output:

```
Expected :laptop, it’s probably
Actual   :laptop, it&rsquo;s probably
```

**Comment by spacecowboy** [comment]
I should also mention that I'm using version 1.x because I can't upgrade kotlin yet (Android restrictions)

**Comment by MohamedRejeb** [comment]
The problem is fixed in 0.2.1, I will try to publish a new release 0.1.5 for kotlin 1.8.x
